python/lsst/dm/rucio/register/resource_bundle.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of dm_rucio_register
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+
+__all__ = ["ResourceBundle"]
+
+logger = logging.getLogger(__name__)
+
+
+class ResourceBundle:
```

Comment:
Consider a pydantic model for this with `did` also being a model.

Reply:
Fixed